app/views/content/is-teaching-right-for-me/languages/_article.html.erb

```diff
@@ -0,0 +1,128 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>
+      A modern languages teacheer opens young minds to the languages and cultures around the world. </p>
+      <p>As we strive to achieve our vision for a 'Global Britain' mutual inter-cultural understanding and insights through the fostering of a shared language, is vital.</p>
+      <p>Qualifications are available in 18 modern languages with the must popular being French, German and Spanish in key stage 2 and 3.</p>
```

Comment:
@caitlin-reidy can we give some examples, similar to the other pages of jobs pupils might be able to go into if they study modern languages to show the impact teaching them can have on pupils' lives.